apps/glusterfs/pendingop_entry.go

```diff
@@ -0,0 +1,265 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"bytes"
+	"encoding/gob"
+	"time"
+
+	"github.com/boltdb/bolt"
+	wdb "github.com/heketi/heketi/pkg/db"
+	"github.com/heketi/heketi/pkg/utils"
+	"github.com/lpabon/godbc"
+)
+
+const (
+	NEW_ID                    = ""
+	BOLTDB_BUCKET_PENDING_OPS = "PENDING_OPERATIONS"
+	DB_HAS_PENDING_OPS        = "PENDING_OPERATIONS"
```

Comment:
1. I suggest DB_HAS_PENDING_OPS_BUCKET to make it less alarming. Having pending ops in closed Db should alarm.
2. Would like if const was same as the string for dbattributes.

So suggestion to change from 
DB_HAS_PENDING_OPS        = "PENDING_OPERATIONS"
to
DB_HAS_PENDING_OPS_BUCKET        = "DB_HAS_PENDING_OPS_BUCKET"

Reply:
Updated.